torch/testing/_internal/common_methods_invocations.py

```diff
@@ -12156,6 +12162,36 @@ def sample_inputs_alias_copy(op_info, device, dtype, requires_grad, **kwargs):
                    'test_schema_correctness',
                    dtypes=(torch.complex64, torch.complex128)),
            )),
+    OpInfo('_convert_weight_to_int4pack',
+           dtypes=integral_types(),
```

Comment:
I think changing the following should fix the issue.
```suggestion
           dtypes=(torch.uint8,),
```

Reply:
Update to `dtypes=_dispatch_dtypes((torch.uint8,)),` instead.